src/lib/constants/Headers.ts

```diff
@@ -0,0 +1,6 @@
+export enum Headers {
+  VTEX_ORIGINAL_CREDENTIAL = 'x-vtex-original-credential',
+  VTEX_RUNTIME_API = 'x-vtex-runtime-api',
+  VTEX_STICKY_HOST = 'x-vtex-sticky-host',
+  VTEX_UPSTREAM_TARGET = 'x-vtex-upstream-target',
```

Comment:
Should I add a little doc on each? Like:
https://github.com/vtex/toolbelt/blob/ec41288edb94a5e8b3b527c151228985c73c18b6/src/lib/constants/EnvVariables.ts#L1-L10

Reply:
Sounds good! I'd like that